2020-12-28 16:18:53 INFO: Access Token has expired. Hence refreshing.
2020-12-28 16:18:55 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->ingresar()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-28 16:19:40 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid query formed' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(72)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->ingresar()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-28 16:20:06 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid query formed' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(72)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->ingresar()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-28 16:20:10 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->ingresar()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-28 19:24:29 INFO: Access Token has expired. Hence refreshing.
2020-12-28 19:27:21 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Home.php(34): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Home->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-28 20:12:20 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Marca:equals:32...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Marca:equals:32...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Auto\Cotizar.php(27): App\Libraries\Zoho->searchRecordsByCriteria('Modelos', 'Marca:equals:32...', 2, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Auto\Cotizar->modelos()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auto\Cotizar))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-28 20:27:31 INFO: Access Token has expired. Hence refreshing.
2020-12-29 07:14:30 INFO: Access Token has expired. Hence refreshing.
2020-12-29 07:14:34 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Home.php(34): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Home->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-29 07:18:55 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Home.php(34): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Home->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-29 21:09:22 INFO: Access Token has expired. Hence refreshing.
2020-12-29 21:09:23 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-29 21:09:50 SEVERE: Exception occured while getting OAuthTokens from DB(file::ZohoOAuthPersistenceHandler)(mysqli::__construct(): (HY000/2002): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
)
ErrorException: mysqli::__construct(): (HY000/2002): No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión.
 in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceHandler.php:86
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::__const...', 'C:\\xampp\\htdocs...', 86, Array)
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceHandler.php(86): mysqli->__construct('localhost:3306', 'root', '', 'zohooauth')
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceHandler.php(40): zcrmsdk\oauth\persistence\ZohoOAuthPersistenceHandler->getMysqlConnection()
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\ZohoOAuthClient.php(38): zcrmsdk\oauth\persistence\ZohoOAuthPersistenceHandler->getOAuthTokens('tecnologia@grup...')
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\utility\ZCRMConfigUtil.php(104): zcrmsdk\oauth\ZohoOAuthClient->getAccessToken('tecnologia@grup...')
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(88): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#6 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(138): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#7 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#8 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#9 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#10 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#11 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->post()
#12 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#13 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#15 {main}
2020-12-29 21:09:50 INFO: Access Token has expired. Hence refreshing.
2020-12-29 21:10:04 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-29 21:18:42 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-29 22:09:29 INFO: Access Token has expired. Hence refreshing.
2020-12-29 22:09:29 SEVERE: Exception occured while Saving OAuthTokens to file(file::ZohoOAuthPersistenceByFile)(file_put_contents(http://localhost/portal/public/zcrm_oauthtokens.txt): failed to open stream: HTTP wrapper does not support writeable connections)
ErrorException: file_put_contents(http://localhost/portal/public/zcrm_oauthtokens.txt): failed to open stream: HTTP wrapper does not support writeable connections in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceByFile.php:90
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 90, Array)
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceByFile.php(90): file_put_contents('http://localhos...', 'a:0:{}')
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceByFile.php(23): zcrmsdk\oauth\persistence\ZohoOAuthPersistenceByFile->deleteOAuthTokens('tecnologia@grup...')
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\ZohoOAuthClient.php(97): zcrmsdk\oauth\persistence\ZohoOAuthPersistenceByFile->saveOAuthData(Object(zcrmsdk\oauth\utility\ZohoOAuthTokens))
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\ZohoOAuthClient.php(47): zcrmsdk\oauth\ZohoOAuthClient->refreshAccessToken('1000.6a6daa16fb...', 'tecnologia@grup...')
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\utility\ZCRMConfigUtil.php(104): zcrmsdk\oauth\ZohoOAuthClient->getAccessToken('tecnologia@grup...')
#6 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(88): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#7 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(138): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#8 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#9 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#10 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#11 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#12 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->post()
#13 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#14 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#16 {main}
2020-12-29 22:09:29 SEVERE: Exception occured while Saving OAuthTokens to file(file::ZohoOAuthPersistenceByFile)(file_put_contents(http://localhost/portal/public/zcrm_oauthtokens.txt): failed to open stream: HTTP wrapper does not support writeable connections)
ErrorException: file_put_contents(http://localhost/portal/public/zcrm_oauthtokens.txt): failed to open stream: HTTP wrapper does not support writeable connections in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceByFile.php:90
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 90, Array)
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceByFile.php(90): file_put_contents('http://localhos...', 'a:0:{}')
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\persistence\ZohoOAuthPersistenceByFile.php(23): zcrmsdk\oauth\persistence\ZohoOAuthPersistenceByFile->deleteOAuthTokens('tecnologia@grup...')
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\ZohoOAuthClient.php(97): zcrmsdk\oauth\persistence\ZohoOAuthPersistenceByFile->saveOAuthData(Object(zcrmsdk\oauth\utility\ZohoOAuthTokens))
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\oauth\ZohoOAuthClient.php(47): zcrmsdk\oauth\ZohoOAuthClient->refreshAccessToken('1000.6a6daa16fb...', 'tecnologia@grup...')
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\utility\ZCRMConfigUtil.php(104): zcrmsdk\oauth\ZohoOAuthClient->getAccessToken('tecnologia@grup...')
#6 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(88): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#7 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(138): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#8 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#9 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#10 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#11 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#12 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->post()
#13 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#14 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#16 {main}
2020-12-29 22:10:09 INFO: Access Token has expired. Hence refreshing.
2020-12-29 22:10:10 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Email:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Email:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Login.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Contacts', '((Email:equals:...', 1, 1)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Login->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-31 09:13:50 INFO: Access Token has expired. Hence refreshing.
2020-12-31 09:14:43 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Marca:equals:32...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Marca:equals:32...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Auto\Cotizar.php(27): App\Libraries\Zoho->searchRecordsByCriteria('Modelos', 'Marca:equals:32...', 4, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Auto\Cotizar->modelos()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auto\Cotizar))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2020-12-31 10:18:59 INFO: Access Token has expired. Hence refreshing.
2021-01-01 08:49:51 INFO: Access Token has expired. Hence refreshing.
2021-01-01 11:19:58 INFO: Access Token has expired. Hence refreshing.
2021-01-01 11:20:08 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Marca:equals:32...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Marca:equals:32...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(31): App\Libraries\Zoho->searchRecordsByCriteria('Modelos', 'Marca:equals:32...', 4, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->modelos()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:20:19 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Marca:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Marca:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(65): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Marca:equals:...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:20:19 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Modelo:equals...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Modelo:equals...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(73): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Modelo:equals...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:20:20 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Plan:equals:322...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Plan:equals:322...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(83): App\Libraries\Zoho->searchRecordsByCriteria('Recargos', 'Plan:equals:322...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:21:04 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Marca:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Marca:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(65): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Marca:equals:...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:21:04 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Modelo:equals...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Modelo:equals...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(73): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Modelo:equals...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:21:06 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Plan:equals:322...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Plan:equals:322...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(83): App\Libraries\Zoho->searchRecordsByCriteria('Recargos', 'Plan:equals:322...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:21:09 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Modelo:equals...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Modelo:equals...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(73): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Modelo:equals...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:21:11 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Modelo:equals...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Modelo:equals...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(73): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Modelo:equals...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:21:12 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Plan:equals:322...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Plan:equals:322...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(83): App\Libraries\Zoho->searchRecordsByCriteria('Recargos', 'Plan:equals:322...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 11:21:49 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Home.php(34): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Home->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-01 12:49:09 INFO: Access Token has expired. Hence refreshing.
2021-01-02 18:07:23 INFO: Access Token has expired. Hence refreshing.
2021-01-02 18:19:04 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Marca:equals:32...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Marca:equals:32...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(29): App\Libraries\Zoho->searchRecordsByCriteria('Modelos', 'Marca:equals:32...', 4, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->modelos()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-02 18:19:09 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Marca:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Marca:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(63): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Marca:equals:...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-02 18:19:10 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Modelo:equals...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Modelo:equals...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(71): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Modelo:equals...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-02 18:19:11 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Plan:equals:322...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Plan:equals:322...', Array)
#7 C:\xampp\htdocs\portal\app\Libraries\CalcularPrima.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Recargos', 'Plan:equals:322...', 1, 200)
#8 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(83): App\Libraries\CalcularPrima->auto(Object(App\Libraries\Zoho), '322237300000883...', 'Camioneta', '2020', '322237300000577...', '500000')
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#11 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#13 {main}
2021-01-02 18:19:29 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Marca:equals:...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Marca:equals:...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(63): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Marca:equals:...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-02 18:19:29 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Modelo:equals...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Modelo:equals...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(71): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Modelo:equals...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-02 18:19:30 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Plan:equals:322...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Plan:equals:322...', Array)
#7 C:\xampp\htdocs\portal\app\Libraries\CalcularPrima.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Recargos', 'Plan:equals:322...', 1, 200)
#8 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(83): App\Libraries\CalcularPrima->auto(Object(App\Libraries\Zoho), '322237300000883...', 'Camioneta', '2020', '322237300000577...', '500000')
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#11 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#13 {main}
2021-01-02 18:19:33 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Modelo:equals...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Modelo:equals...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(71): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Modelo:equals...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-02 18:19:35 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', '((Modelo:equals...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('((Modelo:equals...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(71): App\Libraries\Zoho->searchRecordsByCriteria('Restringidos', '((Modelo:equals...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-02 18:19:36 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Plan:equals:322...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Plan:equals:322...', Array)
#7 C:\xampp\htdocs\portal\app\Libraries\CalcularPrima.php(22): App\Libraries\Zoho->searchRecordsByCriteria('Recargos', 'Plan:equals:322...', 1, 200)
#8 C:\xampp\htdocs\portal\app\Controllers\Cotizar\Auto.php(83): App\Libraries\CalcularPrima->auto(Object(App\Libraries\Zoho), '322237300000883...', 'Camioneta', '2020', '322237300000577...', '500000')
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizar\Auto->post()
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizar\Auto))
#11 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#13 {main}
2021-01-02 21:00:12 INFO: Access Token has expired. Hence refreshing.
2021-01-02 22:01:06 INFO: Access Token has expired. Hence refreshing.
2021-01-02 22:38:13 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Home.php(34): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Home->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-02 22:43:52 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Lista.php(24): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Lista->polizas()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lista))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-03 01:52:19 INFO: Access Token has expired. Hence refreshing.
2021-01-03 01:52:23 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Home.php(34): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Home->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-03 18:02:31 INFO: Access Token has expired. Hence refreshing.
2021-01-05 07:41:50 INFO: Access Token has expired. Hence refreshing.
2021-01-06 08:03:46 INFO: Access Token has expired. Hence refreshing.
2021-01-06 09:02:50 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Marca:equals:32...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Marca:equals:32...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Cotizaciones.php(124): App\Libraries\Zoho->searchRecordsByCriteria('Modelos', 'Marca:equals:32...', 2, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Cotizaciones->modelos()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cotizaciones))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-06 09:14:43 INFO: Access Token has expired. Hence refreshing.
2021-01-06 09:30:57 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Home.php(32): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Home->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-06 09:31:02 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Polizas.php(17): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Polizas->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Polizas))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-06 09:44:36 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\APIRequest.php(148): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\api\handler\MassEntityAPIHandler.php(335): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 C:\xampp\htdocs\portal\vendor\zohocrm\php-sdk-archive\src\crm\crud\ZCRMModule.php(1131): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords(Array, 'criteria', 'Account_Name:eq...')
#6 C:\xampp\htdocs\portal\app\Libraries\Zoho.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Account_Name:eq...', Array)
#7 C:\xampp\htdocs\portal\app\Controllers\Home.php(32): App\Libraries\Zoho->searchRecordsByCriteria('Deals', 'Account_Name:eq...', 1, 200)
#8 C:\xampp\htdocs\portal\system\CodeIgniter.php(871): App\Controllers\Home->index()
#9 C:\xampp\htdocs\portal\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\portal\system\CodeIgniter.php(307): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\portal\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
2021-01-06 10:14:45 INFO: Access Token has expired. Hence refreshing.
